kiauh/extensions/telegram_bot/moonraker_telegram_bot.py

```diff
@@ -118,7 +118,7 @@ def _prep_env_file_content(self) -> str:
         )
         env_file_content = env_file_content.replace(
             "%CFG%",
-            f"{self.base.cfg_dir}/printer.cfg",
+            self.base.cfg_dir.joinpath(self.cfg_file).as_posix()
```

Comment:
I think it should rather be this?
```py
self.cfg_file.as_posix()
```
self.cfg_file is defined in the post init method here:
https://github.com/dw-0/kiauh/pull/600/files#diff-865abf821e46fc405feedc03285f84ad46004e8f6a7575847e6c3a004c4abbefR52
and already uses `self.base.cfg_dir`.